MatrixSDK/Crypto/MXOlmDevice.h

```diff
@@ -23,6 +23,8 @@
 #import "MXCryptoStore.h"
 #import "MXDecrypting.h"
 
+@class MXOutboundSessionInfo;
```

Comment:
This is a utility class for `MXMegolmEncryption` for its internal state management, can we let it there?